src/main/java/org/myrobotlab/service/config/ProgramABConfig.java

```diff
@@ -15,5 +15,15 @@ public class ProgramABConfig extends ServiceConfig {
   // search engine - currently wikipedia
   // search for sraix
   public String search;
+  
+  /**
+   * enables global session and sleep/wake functionality
+   */
+  public boolean useGlobalSession;
```

Comment:
you've added this configuration variable, but it isn't actually used anywhere. please remove.